cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaStatusTest.java

```diff
@@ -1153,4 +1189,18 @@ public MockModelWarningsStatusKafkaAssemblyOperator(Vertx vertx, PlatformFeature
         }
 
     }
+
+    class MockClusterIdStatusKafkaAssemblyOperator extends KafkaAssemblyOperator  {
+        public MockClusterIdStatusKafkaAssemblyOperator(Vertx vertx, PlatformFeaturesAvailability pfa, CertManager certManager, PasswordGenerator passwordGenerator, ResourceOperatorSupplier supplier, ClusterOperatorConfig config) {
+            super(vertx, pfa, certManager, passwordGenerator, supplier, config);
+        }
+
+        @Override
+        Future<Void> reconcile(ReconciliationState reconcileState)  {
+            reconcileState.kafkaStatus.setClusterId("CLUSTERID");
+
+            return Future.succeededFuture();
+        }
```

Comment:
Since you now mocked the cluster-id in the Admin provider, I do not think ou need this ... you should really just call the actual `kafkaGetClusterId` method IMHO which should set it on its own.